gson/src/main/java/com/google/gson/GsonBuilder.java

```diff
@@ -488,6 +492,19 @@ public GsonBuilder setPrettyPrinting() {
     return this;
   }
 
+  /**
+   * Configures Gson to output JSON that uses a certain kind of newline.
+   * This option only affects JSON serialization.
+   *
+   * <p>Has no effect if the serialized format is a single line.</p>
+   *
+   * @return a reference to this {@code GsonBuilder} object to fulfill the "Builder" pattern
+   */
```

Comment:
Would probably be good to add `@since $next-version$` here as well.

Reply:
Done